ocaml/xapi/xapi_vm_migrate.ml

```diff
@@ -224,8 +235,12 @@ let infer_vgpu_map ~__context ?remote vm =
     List.map (fun self ->
       let vgpu = XenAPI.VGPU.get_record rpc session self in
       let device = vgpu.API.vGPU_device in
+      let pgpu = vgpu.API.vGPU_scheduled_to_be_resident_on in
+      if not (Db.is_valid_ref __context pgpu) then
```

Comment:
This won't work. Hmm, I don't think we expose 'is_valid_ref' externally - perhaps we could emulate by just doing a get_uuid or something?